ui/src/components/dialog/QasDialog.vue

```diff
@@ -14,7 +14,7 @@
       <section class="text-body1 text-grey-8">
         <component :is="componentTag" ref="form">
           <slot name="description">
-            <div v-if="card.description">{{ card.description }}</div>
+            <component :is="descriptionComponent">{{ card.description }}</component>
```

Comment:
Seguindo o padrão do `is` acima, não seria melhor chamar de `descriptionComponentTag`?

Reply:
feito